src/components/Layout/index.tsx

```diff
@@ -18,8 +22,54 @@ const navBarContent = (title: string, refUrl: string, isActive: boolean = false)
     </Link>
   );
 };
+  
+const { SUCCESS, ERROR } = ToastTypes;
+
+const SELF_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/users/self`;
+
+const Layout: FC<Props> = ({ children, toggleEditButton }) => {
+  const [IsUserAuthorized, setIsUserAuthorized] = useState(false);
+  const [showEditButton, setShowEditButton] = useState(false);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const { requestPromise } = fetch({ url: SELF_URL });
+        const { data } = await requestPromise;
+        const userRoles = {
+          adminUser: data.roles?.admin,
+          superUser: data.roles?.super_user,
+        };
+        const { adminUser, superUser } = userRoles;
+        setIsUserAuthorized(!!adminUser || !!superUser);
+      } catch (err: any) {
+        toast(ERROR, err.message);
+      }
+    };
+    fetchData();
+
+    return () => {
+      setIsUserAuthorized(false);
+    };
+  }, []);
+
+  function handleKeyDown(event: any): void {
+    const alt_key = 18;
```

Comment:
Since this is not a variable name and it is more used for updating users that it is ALT_KEY
Would `ALT_KEY` be better here?